### ISSUE TITLE
Out of date documentation for network controller

### ISSUE THREAD

**Issue by azertyalex** [JimberSoftware/docs#13]
![Image](https://github.com/JimberSoftware/docs/assets/9082988/bcc08fa2-ea29-4888-8c8c-5e648b4ab593)

**Comment by danny-delrue** [comment]
I tried to download Networkcontroller from https://signal.staging.jimber.io/downloads , but the download stucks at 1.058.000 KB. Same result after restarting laptop.

**Comment by danny-delrue** [comment]
Documentation has been updated. Restarting is probably not necessary after all, and there is now another way to start the service?

**Comment by JorisBassle** [comment]
@danny-delrue yes this is a known problem on staging for downloading big files. You can resume the download when it gets stuck. 


![Image](https://github.com/JimberSoftware/docs/assets/36887810/a8dc9bb8-944f-4ea8-b514-87bdb90da429)
